src/bin/identities/CommandList.ts

```diff
@@ -90,25 +89,18 @@ class CommandList extends CommandPolykey {
           for (const gestalt of gestalts) {
             output.push(`gestalt ${count}`);
             output.push(`permissions: ${gestalt.permissions ?? 'None'}`);
-
             // Listing nodes
             for (const node of gestalt.nodes) {
               output.push(`${node.id}`);
             }
             // Listing identities
             for (const identity of gestalt.identities) {
-              output.push(
-                parsers.formatIdentityString(
-                  identity.providerId,
-                  identity.identityId,
-                ),
-              );
+              output.push(`${identity.providerId}:${identity.identityId}`);
             }
             output.push('');
```

Comment:
I can see there's some problems with these where the `retryAuthentication` should just return the stream.

I think this can be addressed in a later PR then. We would move most of that logic our of `retryAuthentication`.